help/de/Help_ch_GUIMenus.xml

```diff
@@ -46,20 +42,20 @@
         <title>Das &app; Hauptfenster</title><mediaobject>
           <imageobject role="html">
```

Comment:
Die Bildcontainer sollten wir vereinheitlichen wie in https://wiki.gnucash.org/wiki/Documentation_Update_Instructions#Indexing. Du darfst aber gerne noch besssere Varianten suchen. Das war nur  irgendwann mal mein bestes Ergebnis.